lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -652,25 +705,23 @@ module.exports = class MethodMappings {
           });
         },
 
-        moveTo(id, xoffset, yoffset) {
-          const data = {};
+        //TODO: update parameters
+        /**
+         *
+         * @param {{async: (boolean|undefined), bridge: (boolean|undefined)}} options
+         * @param {number|undefined} duration
+         * @param {Origin|WebElement|undefined} origin
+         * @param {number|undefined} xOffset
+         * @param {number|undefined} yOffset
```

Comment:
for params that are optional, please use the square brackets, as in: `@param {number} [xOffset]`